plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -904,6 +985,7 @@ public void setColumnType(ConnectorSession session, ConnectorTableHandle table,
     @Override
     public void renameTable(ConnectorSession session, ConnectorTableHandle table, SchemaTableName newTableName)
     {
+        verify(!(table instanceof JdbcProcedureHandle), "Not a table reference: %s", table);
```

Comment:
You don't need this.

Reply:
Can we replicate it for all alter options ?